llvm/test/CMakeLists.txt

```diff
@@ -133,6 +133,18 @@ if(LLVM_BUILD_EXAMPLES)
     )
 endif()
 
+if(LLVM_TOOL_PSTORE_BUILD)
```

Comment:
Unfortunately, I'm not at all familiar with this system: how does this variable (and the others below) get set?

Reply:
They get created automagically by the cmake system for "tools" in the LLVM system (e.g. LLVM_TOOL_CLANG_BUILD etc.). Thinking about it, we probably don't completely support disabling these things (e.g. LLVM_TOOL_PSTORE_BUILD=OFF). Not sure how much we care about that for now.

Reply:
>  we probably don't completely support disabling these things (e.g. LLVM_TOOL_PSTORE_BUILD=OFF)

I'm pretty sure that disabling pstore would cause the build to fail.

> Not sure how much we care about that for now.

Probably not at all for now.